en/ros2/px4_ros2_interface_lib.md

```diff
@@ -193,6 +193,8 @@ The following steps are required to get started:
 The following sections describe specific functionality provided by this library.
 In addition, any other PX4 topic can be subscribed or published.
 
+Detailed API usage can be found here: https://auterion.github.io/px4-ros2-interface-lib/modules.html
```

Comment:
Things are being moved around in https://github.com/PX4/PX4-user_guide/pull/2972

I'd update that PR with this comment and remove from here.